### PR TITLE
wdctl: add page

### PR DESCRIPTION
**Pull request by endorama**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/wdctl.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# wdctl
+
+> Shoes hardware watchdog status.
+> The Linux kernel can reset the system if serious problems are detected. This can be implemented via special watchdog hardware, or via a slightly less reliable software-only watchdog inside the kernel.
+> More information: <https://man7.org/linux/man-pages/man8/wdctl.8.html>.
```
Comment (marchersimon) [inline pages/linux/wdctl.md:5]:
```suggestion
> More information: <https://manned.org/wdctl>.
```

File: pages/linux/wdctl.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# wdctl
+
+> Shoes hardware watchdog status.
```
Comment (marchersimon) [inline pages/linux/wdctl.md:3]:
```suggestion
> Shows the hardware watchdog status.
```

File: pages/linux/wdctl.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+> The Linux kernel can reset the system if serious problems are detected. This can be implemented via special watchdog hardware, or via a slightly less reliable software-only watchdog inside the kernel.
```
Comment (marchersimon) [inline pages/linux/wdctl.md:4]: This doesn't sound important for this command. Can we remove it?

File: pages/linux/wdctl.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+> More information: <https://man7.org/linux/man-pages/man8/wdctl.8.html>.
+
+- Display watchdog status:
```
Comment (marchersimon) [inline pages/linux/wdctl.md:7]:
```suggestion
- Display the watchdog status:
```

File: pages/linux/wdctl.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+`wdctl`
+
+- Display watchdog status on a single line (`key=value` pairs):
```
Comment (marchersimon) [inline pages/linux/wdctl.md:11]:
```suggestion
- Display the watchdog status in a single line in `key=value` pairs:
```

File: pages/linux/wdctl.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+`wdctl --oneline`
+
+- Display only some watchdog flags (list is driver specific):
```
Comment (marchersimon) [inline pages/linux/wdctl.md:15]:
```suggestion
- Display only specific watchdog flags (list is driver specific):
```

File: pages/linux/wdctl.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+`wdctl --flags {{flag list}}`
```
Comment (marchersimon) [inline pages/linux/wdctl.md:17]:
```suggestion
`wdctl --flags {{flag_list}}`
```

File: pages/linux/wdctl.md (hunk 3 above)
Comment (endorama) [reply pages/linux/wdctl.md:4]:
```suggestion
```

**Review by marchersimon — APPROVED** [review]
Thanks @endorama

File: pages/linux/wdctl.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
+# wdctl
+
+> Shows the hardware watchdog status.
```
Comment (CleanMachine1) [inline pages/linux/wdctl.md:3]:
```suggestion
> Show the hardware watchdog status.
```

**Review by CleanMachine1 — APPROVED** [review]

File: pages/linux/wdctl.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://manned.org/wdctl>.
+
+- Display the watchdog status:
+
+`wdctl`
+
+- Display the watchdog status in a single line in `key=value` pairs:
```
Comment (navarroaxel) [inline pages/linux/wdctl.md:10]: I not sure if specific the format in the example description, maybe this is better
```suggestion
- Display the watchdog status in a single line in key-value pairs:
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**